compiler/ir/backend.common/src/org/jetbrains/kotlin/backend/common/lower/loops/HeaderInfo.kt

```diff
@@ -9,8 +9,24 @@ package org.jetbrains.kotlin.backend.common.lower.loops
 
 import org.jetbrains.kotlin.backend.common.CommonBackendContext
 import org.jetbrains.kotlin.backend.common.ir.Symbols
+import org.jetbrains.kotlin.backend.common.lower.loops.handlers.ArrayIndicesHandler
```

Comment:
Why not `import org.jetbrains.kotlin.backend.common.lower.loops.handlers.*`? :)

Reply:
Fixed. I reformatted all the files, I thought that would collapse the imports to `.*`.